### DISCUSSION TITLE
Feature Request - Tap actions for separator card

### DISCUSSION THREAD

**Discussion by Daniel-dev22** [Clooos/Bubble-Card#424] [Feature requests]
It would be nice to have tap actions on the separator card so it could be tapped on to launch a pop-up for example.

**Comment by sayam93** [comment]
This would be a great addition for those of us who use the separator cards for room names on the main dashboard and want to display only some of those devices under the room name on the main display. Tapping a separator that could open a pop containing all the devices in that room instead of just the ones on the main dashboard would be neat!

**Comment by Daniel-dev22** [comment]
> This would be a great addition for those of us who use the separator cards for room names on the main dashboard and want to display only some of those devices under the room name on the main display. Tapping a separator that could open a pop containing all the devices in that room instead of just the ones on the main dashboard would be neat!

This is exactly what I had in mind if this was added.

**Comment by Clooos** [comment]
I'm working on it for the next release 🍻

**Comment by Clooos** [comment]
This is now possible with the sub-buttons in the new v2 beta 🎉

**Reply by sayam93** [reply]
Hi, where can I get the beta? I am using the HACS version and it shows 1.7.3 is the latest.
<img width="559" alt="Screenshot 2024-04-15 at 1 59 39 AM" src="https://github.com/Clooos/Bubble-Card/assets/163408168/10d8a04d-8dfc-4f51-a04a-0ffcaa1a0c02">

**Reply by Clooos** [reply]
I've explained everything in this post on Reddit:

https://www.reddit.com/r/homeassistant/s/e23zrqQnzd

**Comment by EvaldasD21** [comment]
Maybe it would be possible to make tap action work anywhere on separator line, not only on sub-button? Would be an awesome feature when using as minimalist separator line without sub-button.
